src/components/Breadcrumb/Element/BreadcrumbElement.ts

```diff
@@ -0,0 +1,50 @@
+import { LitElement, html, unsafeCSS, css } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import ElementStyle from "./BreadcrumbElement.css?inline";
+
+export type CandyBreadcrumbElementProps = {
+  label: string;
+  last?: boolean;
+};
+
+@customElement("candy-breadcrumb-element")
+export class CandyBreadcrumbElement extends LitElement {
+  static styles = css`${unsafeCSS(ElementStyle)}`;
+
+  @property({ type: String })
+  label = "";
+
+  @property({ type: Boolean })
+  last = false;
```

Comment:
You should rename "last" for "isLast". "is" in front of a boolean variable is a good practice to maintain clear naming convention